README_TEMPLATE.md

```diff
@@ -0,0 +1,65 @@
+# {{project-name}}
+
+## Description
+
+{{project-description}}
+
+### Links
+
+* Staging: http://staging.domain.com/?debug=true
+* Live: http://domain.com/
+
+### Team
+
+* PM Ubilabs: Name <name@ubilabs.net>
+* PM Client: Name <name@domain.com>
+
+## Development
+
+### Prerequisites
+Make sure you have the following tools installed:
+
+For example:
+
+* git
+* npm
+
+### Installation
+
+After cloning the repository, install all dependencies:
+
+For example:
+```sh
+npm install # install new dependencies
+```
+
+### Develop
+
+Run the following command to start the server on localhost:
+
+For example:
+```sh
+npm run server # start the server
+```
+
+### Deploy
+
+To deploy the application, run the following commands:
+
+For example:
+```sh
+npm run build
+npm run deploy
```

Comment:
Most of the times, the build task is called from the deploy task. I would suggest to remove this.